src/snippet/mod.rs

```diff
@@ -148,7 +167,12 @@ fn search_fragments<'a>(
 ///
 /// Takes a vector of `FragmentCandidate`s and the text.
 /// Figures out the best fragment from it and creates a snippet.
-fn select_best_fragment_combination(fragments: &[FragmentCandidate], text: &str) -> Snippet {
+fn select_best_fragment_combination(
+    fragments: &[FragmentCandidate],
+    text: &str,
+    highlighting_prefix: &str,
```

Comment:
the arguments are polution this function.

Can we remove them, add a setter on the `Snippet` object instead, and set the prefix/suffix on after the callsite of this function?

Reply:
Removed and moved to Snippet